docs/users.rst

```diff
@@ -173,14 +175,16 @@ signing_intent
   environment. See :ref:`config.yaml-odcs` section for environment configuration
   details, and full explanation of :ref:`signing-intent`.
 
-**If "compose" section is defined, then "pulp_repos" must be set to true or there
-must be a valid "modules" or "packages" key.  If there is a "modules" key, it
-must have a non-empty list of modules.  The "packages" key, and only the "packages"
+inherit
+  boolean to control whether or not to inherit yum repositories and odcs composes
+  from baseimage build, default false. Inheritance works only for builds
+  which aren't scratch or isolated.
```

Comment:
What is the behavior here when it is set to `true` in a scratch/isolated build? If it is just ignored, maybe we could say something like

> ... from base image build, defaults to false. Scratch and isolated builds do not support iInheritance and `false` is always assumed.